report/validation/thermodesorption_spectra/dark/dark_tds.md

```diff
@@ -0,0 +1,256 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.2
+kernelspec:
+  display_name: vv-festim-report-env
+  language: python
+  name: python3
+---
+
+# Deuterium retention in neutron-damaged tungsten
+
+```{tags} 1D, TDS, trapping, transient
+```
+
++++
+
+This validation case is a thermo-desorption spectrum measurement perfomed by {cite}`dark_modelling_2024`.
+
+Several 0.8 mm thick samples of tungsten were damaged via annealing before being used to perform a TDS measurement.
+
+An ion beam with an incident flux of $5.79 \times 10^{19} \ \mathrm{D \ m^{-2} \ s^{-1}}$ was turned on for $72\mathrm{h}$ with an implantation temperature of $370\mathrm{K}$. The sample then rested for $12\mathrm{h}$ at $295\mathrm{K}$ before beggining the TDS measurement at $300\mathrm{K}$ with a temperature ramp of $0.05\mathrm{K}/s$.
+
+To reproduce this experiment, six traps are needed: 1 intrinsic trap and 5 neutron induced traps.
+The trap densities for the neutron induced traps were fitted by {cite}`dark_modelling_2024` for each _dpa_ amount using FESTIM's `NeutronInducedTrap`.
+
+The damage distribution for the neutron-induced traps is as follows:
+$$
+    f(x) = \frac{1}{1 + \exp{ \frac{\left( x - x_0 \right)}{\Delta x} }}
+$$
+
+Tables with the relevant trap parameters are at the bottom of the page.
+
++++
+
+## FESTIM code
+
+```{code-cell} ipython3
+:tags: [hide-input, hide-output]
+
+# Setup
+import festim as F
+import numpy as np
+import sympy as sp
+import matplotlib.pyplot as plt
+
+# Parameters
+
+
+# Deuterium in Tungsten [35]
```

Comment:
I would avoid hard coding references like this since this index is likely to change. If you want to reference the paper for this diffusivity, please do it in a text block